### PR TITLE
Includes p5.js as a default library. 

### PR DESCRIPTION
**Pull request by demc**
Creates a new library option for p5.js (Processing).

Since Processing relies on importing any functions in the global namespace (by design), those functions and constants have been added to the library's "predefined" list for jshint.

Try snapshot ?snapshot=61ba517e-c300-4154-ac84-dbab2845a23c

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6476,6 +6480,12 @@ node-static@^0.7.9:
     mime "^1.2.9"
     optimist ">=0.3.4"
 
+node@^9.4.0:
+  version "9.4.0"
+  resolved "https://registry.yarnpkg.com/node/-/node-9.4.0.tgz#607459ecd79af0f91e74318124bec0537644cdee"
+  dependencies:
+    node-bin-setup "^1.0.0"
+
```
Comment (outoftime) [inline yarn.lock]: I don’t think this is related to the change in question?

**Comment by pwjablonski** [comment]
So cool @demc love p5!

**Comment by demc** [comment]
Good to go? Can this go into production quickly? Thanks!

**Comment by outoftime** [comment]
@demc good to go! should be in production in about ~20 minutes, modulo various levels of caching

**Pull request merged**